tests/functional/adapter/utils/test_data_types.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from dbt.tests.adapter.utils.data_types.test_type_bigint import BaseTypeBigInt
+from dbt.tests.adapter.utils.data_types.test_type_float import (
+    BaseTypeFloat, seeds__expected_csv as seeds__float_expected_csv
+)
+from dbt.tests.adapter.utils.data_types.test_type_int import (
+    BaseTypeInt, seeds__expected_csv as seeds__int_expected_csv
+)
+from dbt.tests.adapter.utils.data_types.test_type_numeric import BaseTypeNumeric
+from dbt.tests.adapter.utils.data_types.test_type_string import BaseTypeString
+from dbt.tests.adapter.utils.data_types.test_type_timestamp import BaseTypeTimestamp
+
+
+class TestTypeBigInt(BaseTypeBigInt):
+    pass
+
+
+# need to explicitly cast this to avoid it being inferred/loaded as a DOUBLE on Spark
+# in SparkSQL, the two are equivalent for `=` comparison, but distinct for EXCEPT comparison
```

Comment:
That is unfortunate behavior.

Reply:
Agreed — the distinction between `=` and `EXCEPT` doesn't make much sense to me, but I'm sure there is a reason (or was, at some point)